components/esm/loadland.cpp

```diff
@@ -75,6 +75,8 @@ namespace ESM
         mContext = esm.getContext();
 
         mLandData = nullptr;
+        for (int i = 0; i < LAND_GLOBAL_MAP_LOD_SIZE; ++i)
+            mWnam[i] = 0;
```

Comment:
General suggestion to work with containers: use algorithms when possible
```suggestion
        std::fill(std::begin(mWnam), std::end(mWnam), 0);
```
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es1-prefer-the-standard-library-to-other-libraries-and-to-handcrafted-code